help/en/html/tools/Audio.shtml

```diff
@@ -37,24 +49,25 @@
 
       <h4>Audio Table controls</h4>
 
-      <p>Below the Audio Table are two buttons, named "Add
+      <p>At the bottom of the Audio Table screen are buttons "Add
```

Comment:
JMRI has no control over *screens*; the Audio Table is in a *window*.

From the [Apple Style Guide](https://help.apple.com/applestyleguide/#/apsge70df12b) (linked to from Apple's notes to developers on help content): "Use [screen] to refer to what the user looks at while working on a computer or on iPad, iPhone, iPod, or Apple Watch. Don’t use display when you mean screen. On a portable computer or an all-in-one computer, the display is the LCD or CRT device. The screen is the surface the user views."

From [Microsoft's Glossary](https://docs.microsoft.com/en-us/windows/win32/uxguide/glossary#w): "A [window is a] rectangular area on a computer screen in which programs and content appear. A window can be moved, resized, minimized, or closed; it can overlap other windows."

Reply:
Done.  Thanks for reading - and helping me make corrections and improvements rather than introducing errors!  Jerry